x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -378,94 +381,103 @@ export const signalRulesAlertType = ({
           }
           const inputIndex = await getInputIndex(services, version, index);
 
-          const {
-            filters: bucketFilters,
-            searchErrors: previousSearchErrors,
-          } = await getThresholdBucketFilters({
-            indexPattern: [outputIndex],
-            from,
-            to,
-            services,
-            logger,
-            ruleId,
-            bucketByFields: normalizeThresholdField(threshold.field),
-            timestampOverride,
-            buildRuleMessage,
-          });
-
-          const esFilter = await getFilter({
-            type,
-            filters: filters ? filters.concat(bucketFilters) : bucketFilters,
-            language,
-            query,
-            savedId,
-            services,
-            index: inputIndex,
-            lists: exceptionItems ?? [],
-          });
+          for (const tuple of tuples) {
```

Comment:
@marshallmain Was there a reason it was in descending order? Should I just clone and reverse the array for threshold rules?

Reply:
No reason for it, other rule types don't depend on the order of the tuples at the moment but I think it's more intuitive overall if they're ascending so that if a gap is detected the additional rule runs execute in the order they would have without a gap. We can just add `tuples.reverse` in `getRuleRangeTuples`, I don't think we even need to clone it.

Reply:
@marshallmain Done!